ST/32_api_requests_in_cucumber_tests/docs/cucumber.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Step definition in Cucumber
+subtitle: " API automated tests based on scenarios"
+menu: cucumber
+---
+# API requests in Cucumber tests
+
+This is a project used for increase technical skills on API requests in Cucumber tests
+Cucumber can be used to implement automated tests based on scenarios described in your Gherkin feature files.
+
+# Step Arguments
+
+Cucumber extracts the text `48` from the step, converts it to an `int`
+and passes it as an argument to the {{% stepdef-body %}}.
+
+The number of parameters in the {{% stepdef-body %}} has to match the number of {{% expression-parameter %}}s in the expression. (If there is a mismatch, Cucumber will throw an error).
+
+## Data Tables
+
+Data tables from Gherkin can be accessed by using the DataTable object as the last parameter in a step definition.
+This conversion can be done either by Cucumber or manually.
+
+{{% text "java" %}}Depending on the table shape as one of the following collections:{{% /text %}}
```

Comment:
this {{% text "java" %}} appears on  page, is it intended ?
<img width="738" alt="image" src="https://github.com/bodacd/automation-study-materials/assets/5145950/abe7de3b-d12e-4456-8b63-8f71653d857c">